ClrMD.Extensions/ClrMDSession.cs

```diff
@@ -93,27 +87,27 @@ public static ClrMDSession LoadCrashDump(string dumpPath, string dacFile = null)
             return new ClrMDSession(target, dacFile);
         }
 
-        public static ClrMDSession AttachToProcess(string processName, uint millisecondsTimeout = 5000, AttachFlag attachFlag = AttachFlag.Invasive)
+        public static ClrMDSession AttachToProcess(string processName)
         {
             Process p = Process.GetProcessesByName(processName).FirstOrDefault();
 
             if (p == null)
                 throw new ArgumentException("Process not found", "processName");
 
-            return AttachToProcess(p, millisecondsTimeout, attachFlag);
+            return AttachToProcess(p);
         }
 
-        public static ClrMDSession AttachToProcess(int pid, uint millisecondsTimeout = 5000, AttachFlag attachFlag = AttachFlag.Invasive)
+        public static ClrMDSession AttachToProcess(int pid)
```

Comment:
AttachFlag no longer exists? Is there an alternative to attach without freezing the process?